src/me/StevenLawson/TotalFreedomMod/Commands/Command_nightvision.java

```diff
@@ -0,0 +1,33 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import net.md_5.bungee.api.ChatColor;
```

Comment:
Spigot has bungee in it so...... Yeah. That happens


> On May 26, 2015, at 6:42 AM, Ryan <notifications@github.com> wrote:
> 
> In src/me/StevenLawson/TotalFreedomMod/Commands/Command_nightvision.java:
> 
> > @@ -0,0 +1,33 @@
> > +package me.StevenLawson.TotalFreedomMod.Commands;
> > +
> > +import net.md_5.bungee.api.ChatColor;
> Yeah, what you are using here is for a Bungee, not sure how that import even came up in all honesty because you shouldnt have a Bungee library at all.
> 
> —
> Reply to this email directly or view it on GitHub.
>